.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,71 @@
+---
+name: "🐛 Bug Report"
+description: Create a report to help us improve
+title: "(short issue description)"
+labels: [bug]
+assignees: []
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: What is the problem? A clear and concise description of the bug.
+    validations:
+      required: true
+
+  - type: textarea
+    id: current
+    attributes:
+      label: Current Behavior
+      description: |
+        Please include full errors, uncaught exceptions, screenshots, and relevant logs.
+        Using environment variable JFROG_CLI_LOG_LEVEL="DEBUG" upon running the command will provide more log information.
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction Steps
+      description: |
+        Provide steps to reproduce the behavior.
+    validations:
+      required: false
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected Behavior
+      description: |
+        What did you expect to happen?
+    validations:
+      required: false
+
+  - type: input
+    id: cli-version
+    attributes:
+      label: JFrog CLI version
+      description: using "jf --version"
+    validations:
+      required: false
+
+  - type: input
+    id: os-version
+    attributes:
+      label: OS version
+    validations:
+      required: false
```

This information is easy to test and may be critical for us in some cases.
```suggestion
      required: true
```